src/fides/cli/__init__.py

```diff
@@ -109,10 +109,6 @@ def cli(ctx: Context, config_path: str, local: bool) -> None:
     if not command:
         echo(cli.get_help(ctx))
 
-    # Check the server health and version if an API command is invoked
-    if command in SERVER_CHECK_COMMAND_NAMES:
```

Comment:
nit: we can probably remove this constant now right?

Reply:
yup, removed now https://github.com/ethyca/fides/pull/5243/commits/f43a2039c7a301c44b912d95f4fb2d8fc37e1420